modules/social_features/social_profile/modules/social_profile_fields/src/Form/SocialProfileFieldsSettingsForm.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+namespace Drupal\social_profile_fields\Form;
+
+use Drupal\Core\Cache\Cache;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Database\Connection;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\field\FieldConfigStorage;
+use Drupal\profile\Entity\ProfileType;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Configure social profile settings.
+ */
+class SocialProfileFieldsSettingsForm extends ConfigFormBase implements ContainerInjectionInterface {
+
+  /**
+   * Field storage.
+   *
+   * @var \Drupal\field\FieldConfigStorage
+   */
+  protected $fieldStorage;
+
+  /**
+   * The database.
+   *
+   * @var \Drupal\Core\Database\Connection
+   */
+  protected $database;
+
+  /**
+   * SocialProfileSettingsForm constructor.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The config factory.
+   * @param \Drupal\field\FieldConfigStorage $field_storage
+   *   Fieldstorage for the profile fields.
+   * @param \Drupal\Core\Database\Connection $database
+   *   Database connection for invalidating caches.
+   */
+  public function __construct(ConfigFactoryInterface $config_factory, FieldConfigStorage $field_storage, Connection $database) {
+    parent::__construct($config_factory);
+    $this->fieldStorage = $field_storage;
+    $this->database = $database;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('config.factory'),
+      $container->get('entity.manager')->getStorage('field_config'),
+      $container->get('database')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'social_profile_admin_settings_form';
```

Comment:
@jochemvn Have you seen that this form conflicts with existing `SocialProfileSettingsForm`?
https://github.com/goalgorilla/open_social/blob/8.x-1.x/modules/social_features/social_profile/src/Form/SocialProfileSettingsForm.php

Reply:
@ribel ,Yes... this PR is reverted here: https://github.com/goalgorilla/open_social/pull/839
The correct one is here: https://github.com/goalgorilla/open_social/pull/787
in that one the form_id is fixed